package.json

```diff
@@ -24,5 +24,6 @@
     "test:release-script": "mocha scripts/test/tests.js",
     "prepare": "pnpm husky install"
   },
-  "type": "module"
+  "type": "module",
+  "packageManager": "pnpm@9.2.0"
```

Comment:
Can you please update version to 9.3.0 and then be free to use pnpm v9.3.0 and run pnpm i.